lib/devise_two_factor/spec_helpers/two_factor_authenticatable_shared_examples.rb

```diff
@@ -77,17 +77,17 @@
     end
 
     it 'validates an OTP within the allowed drift' do
-      otp = ROTP::TOTP.new(otp_secret).at(Time.now + subject.class.otp_allowed_drift, true)
+      otp = ROTP::TOTP.new(otp_secret).at(Time.now + subject.class.otp_allowed_drift)
       expect(subject.validate_and_consume_otp!(otp)).to be true
     end
 
     it 'does not validate an OTP above the allowed drift' do
-      otp = ROTP::TOTP.new(otp_secret).at(Time.now + subject.class.otp_allowed_drift * 2, true)
+      otp = ROTP::TOTP.new(otp_secret).at(Time.now + subject.class.otp_allowed_drift * 2)
       expect(subject.validate_and_consume_otp!(otp)).to be false
     end
 
     it 'does not validate an OTP below the allowed drift' do
-      otp = ROTP::TOTP.new(otp_secret).at(Time.now - subject.class.otp_allowed_drift * 2, true)
+      otp = ROTP::TOTP.new(otp_secret).at(Time.now - subject.class.otp_allowed_drift * 2)
       expect(subject.validate_and_consume_otp!(otp)).to be false
     end
```

Comment:
Hi, line 95-100 we need to fix broken shared example too, now it has escaped email in generated code. So pattern checked needs to include `CGI.escape` of the account (fake email).

Reply:
```
describe '#otp_provisioning_uri' do
    let(:otp_secret_length) { subject.class.otp_secret_length }
    let(:account)           { Faker::Internet.email }
    let(:issuer)            { "Tinfoil" }

    it "should return uri with specified account" do
      expect(subject.otp_provisioning_uri(account)).to match(%r{otpauth://totp/#{CGI.escape account}\?secret=\w{#{otp_secret_length}}})
    end

    it 'should return uri with issuer option', :aggregate_failures do
      expect(subject.otp_provisioning_uri(account, issuer: issuer)).
        to match(%r{otpauth://totp/#{CGI.escape issuer}:#{CGI.escape account}\?.*secret=\w{#{otp_secret_length}}(&|$)})
    end
  end
```